Add organization documentation (#590)
wso2/docs-choreo-dev: en/docs/developer-portal/test-api.md

# Test API
 
 Before using your API in production, you can test it using the integrated OpenAPI console in Choreo. Once the API has is deployed in the required environment, an API consumer can use the application credentials, generate an OAuth 2.0 token, and invoke the API to test it's functionality by providing values for the required parameters. 

This guide will take you through the steps to test an API in Choreo. 

> 1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click  Developer Portal link on the Choreo console header. 

Reviewer: ```suggestion
1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). 

Alternatively, click **Developer Portal** on the Choreo Console header.
```

Author: Same as above

Reviewer: As suggested, the "console" should change to "Console".

    ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-small}
2. Click **APIs**.

> 3. Select an API you want to test and click **Try Out** on the card. In this case, skip step 4. Alterntively, you can click on the API and follow step 4. 

Reviewer: ```suggestion
3. Select an API you want to test and click **Try Out** on the card. In this case, skip step 4. Alternatively, you can click on the API and follow step 4.
```

> 4. Click **Try Out** from the left panel.

Reviewer: ```suggestion
4. In the left pane, click **Try Out**.
```

Author: Fixed.

> 5. Select the required environment from the **Environment** dropdown. 

Reviewer: Shall we use "list" instead of "dropdown" here to comply with the latest practices we agreed on when updating the style guide?

Author: done

Reviewer: This does not appear changed.

>     ![Test Panel](./assets/img/developer-portal/test-api/test-panel.png)

Reviewer: This image is not rendering.

Reviewer: Here too, we need to change "dropdown" to "list".

> 6. Select the relevant**Subscribed Application** from the dropdown that you subscribed the API to. 

Reviewer: ```suggestion
6. From the **Subscribed Application** list, select an application you want to trigger when testing the API.

     !!! info
         This list is visible only if at least one application has subscribed to your API. For more information, see [Manage Subscriptions](manage-subscription/).
```
Suggesting this revision because this way, it is not clear what the name of the dropdown is. Also, refer [1].

Also, a user who reads this guide before reading [Manage Subscriptions](../manage-subscription/) can get confused when he/she does not see this field.

[1]  https://developers.google.com/style/ui-elements#refer-to-ui-elements

Author: triggering the application is incorrect isn't it?

Reviewer: "Invoke" then?

Author: No.. you don't invoke an application. You invoke the API.

Reviewer: Anyway, shall we mention something which make it clear what makes the specific application you select the "relevant" one?

Should it be "From the **Subscribed Application** list, select an application you want to use when testing the API."?

Reviewer: There is also a missing space in this text.

Author: done

7. Click **Get Test Key** on the **Access Token** field to generate a test access token. 

> 8. Next, expand a resource you want to test by clicking on the resource.

Reviewer: ```suggestion
Next, expand a resource you want to test by clicking on it.
```

Reviewer: This still appears the same.

>     ![Test Resource](./assets/img/developer-portal/test-api/test-resource.png)

Reviewer: This image is not rendering.

9. Click **Try it Out** to enable the resource to be tested.  
10. Fill in values for the required parameters and click **Execute**.

>    ![Test Resource](./assets/img/developer-portal/test-api/test-enabled-resource.png)

Reviewer: This image is not rendering.

Reviewer: Now the image is not aligned.
